voter/src/test/java/org/terracotta/voter/ActiveVoterTest.java

```diff
@@ -255,7 +257,7 @@ public void testReregistrationWhenAllStaticHostPortsNotAvailable() throws Timeou
     promote(passiveManager);
 
     waitForLogMessage("Vote owner state: ACTIVE-COORDINATOR");
-
+    status.get().awaitRegistrationWithAll();
```

Comment:
why do we need this?

Reply:
I was mis-interpreting a test failure.  Removed